packages/Pill/src/Pill.styles.js

```diff
@@ -0,0 +1,76 @@
+import { css } from "styled-components";
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+
+const PillColorStyles = {
+  "#d7d7d7": css`
+    color: ${tokens.color.black};
+
+    svg {
+      fill: ${tokens.color.black};
+    }
+  `,
+
+  noRisk: css`
+    background: ${tokens.color.blackLighten60};
+    color: ${tokens.color.black};
+  `,
+
+  lowRisk: css`
+    background: #299a7a;
+  `,
+
+  mediumRisk: css`
+    background: #c9af28;
+  `,
+
+  highRisk: css`
+    background: #cd3c44;
+  `,
+};
+
+const PillSizeStyles = {
+  small: css`
+    border-radius: ${tokens.space} / 50%;
```

Comment:
Sorry, I actually wasn't thinking of using `space` here 😄 I was just recalling we had some tokens for border radius sizes, and wondering whether these cases deserve tokens or not. Up to you anyway 👍

Reply:
Oooh, yeah, i would need to use "button border radius" for 4px. I think this is fine :)

Reply:
Tokens _are_ basically just variables 😄 
So I do think it is a good idea to add a new `pill--radius` token.